soc_062_internal_displacement/README.md

```diff
@@ -1,9 +1,13 @@
 ## Annual Internal Displacement from Natural Disasters Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Raw Incident Data dataset](https://incidentnews.noaa.gov/) for [display on Resource Watch](https://resourcewatch.org/data/explore/soc062c-Internal-Displacement-NRT).
 
-{Describe how the original data came from the source.}
+This dataset was provided by the source as a JSON file. This JSON was transformed into a table so that it could be uploaded to Carto. In order to transform the data from the JSON to the data table used by Resource Watch, the following changes were made:
+- The latitude and longitude features from the JSON were used to create the geometry shown on Resource Watch.
+- Date of the events were obtained from the 'year' feature and the 'displacement_date' feature of the json. This was converted to a datetime object.
+- Description of the events were collected from the 'standard_popup_text' feature.
+- A unique ID for each event was created based on the 'id' feature of the json. This was stored in a new column called 'uid'.
 
-{Describe the steps used to process the data, e.g., "convert variable X from the original netcdf file to a tif to upload to Google Earth Engine."}
+The data shown on Resource Watch can be found in the 'displacement_type' feature of the JSON.
```

Comment:
We also show the size of the displacement...I'm just going to remove this, rather than trying to make sure we always describe exactly what we show on RW.
```suggestion
```